data/compatible_products.yml

```diff
@@ -27,10 +27,10 @@ sets:
 
     - name: Open Liberty
```

Comment:
Going to make a general comment for this whole page...  We need the data to be consistent for all of the compatible products -- versions, links, etc.